tools/stringify/_function.scss

```diff
@@ -4,12 +4,27 @@ section: Tools
 
 Converts a number to a string.
 
+**Use**
 ```scss
 .my-class {
   prop: stringify(<number>);
 }
 ```
 
+**Use with a prop**
+```scss
+.my-class {
+  content: stringify(14);
+}
+```
+
+**CSS Output**
+```css
+.my-class {
+  content: '14';
```

Comment:
Content was the only property I could think of where this might be used, but is there a better one?